[update] Metrics-logging (#2371)
timescale/docs: use-timescale/metrics-logging/integrations.md

1.  Navigate to `Services`. Click on the service you want to connect to your
    exporter.
1.  Navigate to `Operations`, then `Integrations`.
1.  Select an exporter and click `Attach exporter`.

<Highlight type="warning">

> If you would like to attach a logs exporter to an
> already existing service, you have to restart the service only for the
> first logs exporter to be attached. You can attach the logs exporter to the
> service without a need to restart the service.

Reviewer: ```suggestion
If you would like to attach a logs exporter to an already existing 
service, you do not need to restart the service. The service only 
needs to be restarted when you attach the first logs exporter.
```
